README.md

```diff
@@ -417,7 +417,13 @@ Commit the result.
     docker tag cpmonitor:latest cpmonitor:${DATESTR}
     docker tag klimaschutzmonitor-dbeaver:latest klimaschutzmonitor-dbeaver:${DATESTR}
     ```
-9. Stop the server, apply the migrations, start the server:
+9. Stop the reverse proxy:
+    ```sh
+    cd docker/reverseproxy
```

Comment:
Where did this directory come from? Should it also be copied during the `scp` command above and updated here?

Reply:
I am always thinking in terms of the Repo (still can't get my head around the idea that we do not have it there). Actually I don't know how Felix (will) set it up on the server.
Can you correct this @fblampe ?